TappNetwork/home-visiting-app-web
# Cu 2rd5t91 resource directory

# Details

This PR encompasses front and back end changes for the HV App. The resources are filterable by favorites, tags, and construct and are searchable. Users can favorite articles from the directory. The ability to share resources with other users will be added in a future PR.

# Steps to test changes

1. Pull down both branches and run migrations seeder in back end
2. Login in to front end app
3. Go to resource center 
4. Favorite and unfavorite resources.
5. Reload page to ensure persistence on the back end
6. Test all filters
7. Resize to mobile to test responsiveness 

# Issues Addressed

[2rd5t91](https://app.clickup.com/t/2rd5t91)

# Screenshots

<img width="984" alt="Screen Shot 2022-08-24 at 9 09 48 AM" src="https://user-images.githubusercontent.com/29612767/186456410-1b70855a-8fc5-4233-b87d-4b7daec5c1e2.png">
